docs/guide/extending_page.md

```diff
@@ -1,4 +1,4 @@
-# Extending the Page
+# Creating custom Page blocks
 
 !!! enterprise
 
```

Comment:
You can now get rid of the second heading (can't leave a comment at the proper line)